.evergreen/config.yml

```diff
@@ -1702,6 +1702,32 @@ axes:
           GO_DIST: "/opt/golang/go1.15"
           PYTHON3_BINARY: python3
 
+  # OSes that require >= 4.0 for SSL
+  - id: os-ssl-40
+    display_name: OS
+    values:
+      - id: "windows-64-go-1-15"
```

Comment:
Is it possible to use YAML anchor syntax to deduplicate some of this information? IIUC the Windows and macos definitions are the same as those in `os-ssl-32` and only the Ubuntu definition has changed. Given that this is blocking another ticket, I can also understand leaving it as-is for now and filing a new ticket to investigate deduplication across the config. Thoughts?

Reply:
Since there are no usages of YAML anchors or variables across the `config`, I think it's a good idea to clean it up as part of a larger ticket [GODRIVER-1959](https://jira.mongodb.org/browse/GODRIVER-1959).